### PR TITLE
dm: improve binlog schema description

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran.alex@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/9107
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-manage-schema.md [hunk 1]
```diff
@@ -7,30 +7,38 @@ summary: Learn how to manage the schema of the table to be migrated in DM.
...
+
 ## Implementation principles
 
-When you migrate tables using DM, DM performs the following operations on the table schema:
+The internal table schema comes from the following sources:
 
-- For full export and import, DM directly exports the upstream table schema of the current time to SQL files and applies the table schema to the downstream.
+1. For full data migration (`task-mode=all`), the migration task goes through three stages: dump/load/sync, which means full export, full import, and incremental replication. In the dump stage, DM exports the table schema information along with the data and automatically creates the corresponding table in the downstream. In the sync stage, this table schema is used as the starting table scheme for incremental replication.
+2. In the sync stage, when DM handles DDL statements such as `ALTER TABLE`, it updates the internal table schema at the same time.
+3. If the task is an incremental migration (`task-mode=incremental`), in which the downstream has completed creating the table to be migrated, DM obtains the table schema information from the downstream database. This behavior varies with DM versions.
```
Comment (shichun-0415) [inline dm/dm-manage-schema.md:20]:
```suggestion
- For full data migration (`task-mode=all`), the migration task goes through three stages: dump/load/sync, which means full export, full import, and incremental replication. In the dump stage, DM exports the table schema information along with the data and automatically creates the corresponding table in the downstream. In the sync stage, this table schema is used as the starting table scheme for incremental replication.
- In the sync stage, when DM handles DDL statements such as `ALTER TABLE`, it updates the internal table schema at the same time.
- If the task is an incremental migration (`task-mode=incremental`), in which the downstream has completed creating the table to be migrated, DM obtains the table schema information from the downstream database. This behavior varies with DM versions.
```

File: dm/dm-manage-schema.md [hunk 2]
```diff
@@ -7,30 +7,38 @@ summary: Learn how to manage the schema of the table to be migrated in DM.
...
 
-- For incremental replication, the whole data link contains the following table schemas, which might be the same or different:
+For incremental replication, the schema maintenance is complicated. The whole data link contains the following table schemas, which might be the same or different:
```
Comment (shichun-0415) [inline dm/dm-manage-schema.md:22]: This sentence is a little confusing:
`The whole data link contains the following table schemas, which might be the same or different`. 
I think it is intended to say that the data link may include the following table schemas or include other table schemas.
```suggestion
For incremental replication, schema maintenance is complicated. The whole data link contains the following table schemas, which might be the same or different:
```

File: dm/dm-manage-schema.md [hunk 3]
```diff
@@ -7,30 +7,38 @@ summary: Learn how to manage the schema of the table to be migrated in DM.
...
-    * The table schema of the binlog event currently being consumed by DM, identified as `schema-B`. This schema corresponds to the upstream table schema at a historical time.
-    * The table schema currently maintained in DM (the schema tracker component), identified as `schema-I`.
-    * The table schema in the downstream TiDB cluster, identified as `schema-D`.
+* The upstream table schema at the current time, identified as `schema-U`.
+* The table schema of the binlog event currently being consumed by DM, identified as `schema-B`. This schema corresponds to the upstream table schema at a historical time.
+* The table schema currently maintained in DM (the schema tracker component), identified as `schema-I`.
+* The table schema in the downstream TiDB cluster, identified as `schema-D`.
 
-    In most cases, the four table schemas above are the same.
+In most cases, the four table schemas above are the same.
```
Comment (shichun-0415) [inline dm/dm-manage-schema.md:31]: Reason: `same` emphasizes that schemas are the same, from structure to content. `一致` emphasizes data information is identical.
```suggestion
In most cases, the preceding four table schemas are consistent.
```

File: dm/dm-manage-schema.md (hunk 2 above)
Comment (ran-huang) [reply dm/dm-manage-schema.md:22]: You're right about the sentence being confusing...

However, it doesn't look like there are any other table schemas involved. 

What the author tries to say is that **the four table schemas in the diagram** (schema-U, schema-B, schema-I, schema-D) **are usually consistent with one another** (see L31), but **sometimes might be inconsistent** (see L35).

I try to rephrase this sentence, PTAL:

```suggestion
For incremental replication, schema maintenance is complicated. During the whole data replication, the following four table schemas are involved. These schemas might be the consistent or inconsistent with one another:
```

Comment (shichun-0415) [reply dm/dm-manage-schema.md:22]: LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: dm/dm-manage-schema.md (hunk 2 above)
Comment (Frank945946) [reply dm/dm-manage-schema.md:22]: > You're right about the sentence being confusing...
> 
> However, it doesn't look like there are any other table schemas involved.
> 
> What the author tries to say is that **the four table schemas in the diagram** (schema-U, schema-B, schema-I, schema-D) **are usually consistent with one another** (see L31), but **sometimes might be inconsistent** (see L35).
> 
> I try to rephrase this sentence, PTAL:

LGTM

Comment (Frank945946) [reply dm/dm-manage-schema.md:22]: LGTM

**Pull request merged**